pkg/controller/backup_controller.go

```diff
@@ -207,6 +222,7 @@ func (c *backupController) processBackup(key string) error {
 	backupScheduleName := backup.GetLabels()["ark-schedule"]
 	c.metrics.RegisterBackupAttempt(backupScheduleName)
 
+	// TODO: add VSLs to the actual backup
```

Comment:
per previous comment, I don't think we actually need to do this in the controller